src/Core/src/Core/IVisualTreeHelper.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace Microsoft.Maui
+{
+	public interface IVisualTreeHelper
```

Comment:
`IVisualTreeElement`
`IVisualTreeElement? GetVisualParent()`
`IReadOnly<IVisualTreeElement> GetVisualChildren`

Reply:
Added in!